en/guide/define_xml_element.md

```diff
@@ -165,11 +165,20 @@ The main rules for messages are:
 - There may be no more than 64 fields.
 - The `<wip/>` tag may be added to messages that are still being tested.
 - Fields:
-  - must have unique `name`s within a message.
-  - *should* have a description.
-  - *should* use the `units` attribute rather than including units in the description. 
-    Each field should only have **one** or no units.
-  - *should* use the `enum` attribute where possible results are finite/well understood.
+  - `name`: required
+    - must be unique within a message.
```

Comment:
Do we have any other requirements on names? Ie do we prefer them to be snake cased (underscores). Do we prefer full names or do we allow truncation - eg. bat_status, vs battery_status etc?